app/views/liquids/_open_graph.html.erb

```diff
@@ -22,9 +22,9 @@
         <% if page.images.favicon %>
           <img
             alt="favicon"
-            class="m-0 mr-2 radius-0"
-            src="<%= Images::Optimizer.call(page.images.favicon, width: 48) %>"
-            width="24" height="24" loading="lazy" />
+            class="c-embed__favicon m-0 mr-2 radius-0"
+            src="<%= page.images.favicon %>"
```

Comment:
Just to put this change in English, rather than trying to optimize/resize/host the favicon (which was broken) we're serving the img tag with a link to the original here (let the remote site handle this)

I see why this would work.